qiskit/ignis/experiments/calibration/cal_metadata.py

```diff
@@ -36,6 +36,26 @@ class CalibrationMetadata:
     # This mapping is automatically generated at BaseCalibrationExperiment class.
     register_map: Dict[int, int] = None
 
+    def __init__(self,
```

Comment:
you don't need to create full init method because this is automatically generated by dataclass.

please use `__post_init__` instead
https://stackoverflow.com/questions/54863458/force-type-conversion-in-python-dataclass-init-method

Reply:
Thanks for pointing out `__init_post__`. I have amended accordingly.